keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ArtifactSummary.kt

```diff
@@ -57,6 +57,7 @@ data class ArtifactSummaryInEnvironment(
   val deployedAt: Instant? = null,
   val replacedAt: Instant? = null,
   val replacedBy: String? = null,
```

Comment:
Adding a field to the artifact summary to indicate whether this version is pinned in this environment

Reply:
Also, now, information about who pinned it and when they pinned it